integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/sort/SearchSortIT.java

```diff
@@ -397,6 +397,46 @@ public void toAbsolutePath_withRoot_null() {
 				.hasMessageContaining( "'relativeFieldPath' must not be null" );
 	}
 
+	@Test
+	public void missingHighest() {
+		StubMappingScope scope = mainIndex.createScope();
+
+		assertThatQuery(
+				scope.query()
+						.where( f -> f.matchAll() )
+						.sort( s -> s.field( "string" ).asc().missing().highest() )
+						.toQuery()
+		).hasDocRefHitsExactOrder( mainIndex.typeName(), FIRST_ID, SECOND_ID, THIRD_ID, EMPTY_ID );
+
+
+		assertThatQuery(
+				scope.query()
+						.where( f -> f.matchAll() )
+						.sort( s -> s.field( "string" ).desc().missing().highest() )
+						.toQuery()
+		).hasDocRefHitsExactOrder( mainIndex.typeName(), EMPTY_ID, THIRD_ID, SECOND_ID, FIRST_ID );
+	}
+
+	@Test
+	public void missingLowest() {
+		StubMappingScope scope = mainIndex.createScope();
+
+		assertThatQuery(
+				scope.query()
+						.where( f -> f.matchAll() )
+						.sort( s -> s.field( "string" ).asc().missing().lowest() )
+						.toQuery()
+		).hasDocRefHitsExactOrder( mainIndex.typeName(), EMPTY_ID, FIRST_ID, SECOND_ID, THIRD_ID );
+
+
+		assertThatQuery(
+				scope.query()
+						.where( f -> f.matchAll() )
+						.sort( s -> s.field( "string" ).desc().missing().lowest() )
+						.toQuery()
+		).hasDocRefHitsExactOrder( mainIndex.typeName(), THIRD_ID, SECOND_ID, FIRST_ID, EMPTY_ID );
+	}
+
```

Comment:
I don't get why these tests are necessary... this is already tested elsewhere.

I don't know why other, pre-existing methods in this class rely on `.missing().last()`, but I doubt they're actually intended to test this feature. More likely these tests were introduced back when there was no consistent default behavior for missing values across backends.